scripts/wsrep_sst_xtrabackup-v2.sh

```diff
@@ -1371,19 +1378,39 @@ recv_data_from_donor_to_joiner()
     if [[ ${#RC[@]} -lt 1 ]]; then
         wsrep_log_error "******************* FATAL ERROR ********************** "
         wsrep_log_error "SST script interrupted"
-        wsrep_log_error "******************* FATAL ERROR ********************** "
+        wsrep_log_error "****************************************************** "
         exit 32
     fi
 
     if [[ ${RC[0]} -eq 124 ]]; then
         wsrep_log_error "******************* FATAL ERROR ********************** "
         wsrep_log_error "Possible timeout in receving first data from donor in gtid/keyring stage"
+        wsrep_log_error "This message is related to sst-initial-timeout/joiner-timeout configuration variable."
+        wsrep_log_error "It's current value is $WSREP_SST_JOINER_TIMEOUT seconds."
+        wsrep_log_error "This message means that SST transfer didn't start in $WSREP_SST_JOINER_TIMEOUT seconds"
+        wsrep_log_error "after requesting it."
+        wsrep_log_error "The most probable reason is broken network connectivity between Donor and Joiner (this) node."
+        wsrep_log_error "Check the network connection and restart the Joiner node."
```

Comment:
Can we rephrase it to something like?

_The SST process on the donor did not start in the last ${WSREP_SST_JOINER_TIMEOUT} seconds (defined by the sst-initial-timeout variable) after requesting for the state transfer. This could be due to a broken network connectivity between the donor and the joiner (this node). Consider checking the network connection and restarting the Joiner node._

@patrickbirch Any suggestions?